packages/browser-wallet-api-types/README.md

```diff
@@ -1 +1,84 @@
 # browser-wallet-api-types
+
+This package includes the types for the API to be used in web applications for communicating with the Concordium browser wallet.
+
+This package changes the type of the window object, to include the injected api that is available on `window.concordium`.
+
+Note that this package only contains the types and a description of the API, which is rejected by the Concordium browser wallet.
+
+The actual implementation can be found [In the Concordium browser wallet repository.](https://github.com/Concordium/concordium-browser-wallet/tree/main/packages/browser-wallet-api)
+
+## Using the API
```

Comment:
I think we should maybe have a section on including the types for the window object? You can have a look at how it's set up in the `tsconfig.json` of the piggybank example.